chat-plugins/wheels.js

```diff
@@ -0,0 +1,107 @@
+    }
+    endDisplay() {
+        return '|raw|<div style="padding: 5px; background-color: #FFFFCC; color: #FFB41F; border: solid, 1px, #FFB41F; font-size: 15px;">The wheel has landed on: <em><strong>' + this.wheelValue + '</strong></em><br>Positive Numbers = Host wins<br /> Negative Numbers = Spinner wins</div>'
+    }
+ };
+ const wheels = {
+    scrubs: [-5,-4,-3,-2,-2,-1,-1,-1,-1,-1,-1,+1,+1,+3,+3,+3,+3,+3,+3,+5],
+    wealth: [-200,-100,-100,-50,-50,-25,-25,-25,+50,+200,+200,+300,+300,+400],
+    diversity: [-40,-30,-20,-10,-15,-4,-2,-1,+2,+10,+30,+50,+50,+50],
+    balance: [-30,-20,-10,-5,-5,-5,-5,-2,-1,+3,+3,+15,+15,+30,+30,+30],
+    death; [-50,-5,-5,-1,-1,-1,-1,-1,-1,-1,-1,+1,+1,+1,+1,+10,+10,+50]
+ };
+ 
+function spinWheel(wheel) {
+    return wheel[Math.floor(Math.random() * wheel.length)];
+};
+
+function isMoney(money) {
+	let numMoney = Number(money);
+	if (isNaN(money)) return "Must be a number.";
+	if (String(money).includes('.')) return "Cannot contain a decimal.";
+	if (numMoney < 1) return "Cannot be less than one buck.";
+	return numMoney;
+}
+
+function checkMoney(money) {
+	let check = true;
+	let numMoney = Number(money)
+	if (isNaN(numMoney)) check = false;
+	if (String(money).includes('.')) check = false;
+	if (numMoney < 1) check = false;
+	return check;
+}
+
+exports.commands = {
+    wheel: {
+        create: function(target, room, user) {
+            if (!target || target.indexOf(',') < 0) return this.parse('/wheel help');
+            let targets = target.split(",");
+            if (room.wheel) return this.errorReply('There is already a wheel active in this room.');
+            if (!targets[1]) return this.parse('/wheel help');
+            if (!wheels[targets[0]]) return this.errorReply('The wheel specifed does not exist.');
+            Economy.readMoney(user.userid, money => {
+            	if (money < Math.abs(wheels[targets[0]][wheels[targets[0]].length]) * Number(targets[1])) return this.errorReply('You do not have enough bucks to create this game.');
+            	if(!checkMoney(targets[1])) return this.errorReply('Multipliers ' + isMoney(targets[1]) + '.');
+            	room.wheel = new Wheel(parseInt(targets[1]), user.userid, wheels[targets[0]], spinWheel(wheels[targets[0]]));
+            	room.add(room.wheel.initDisplay(target)); //make look fancy later
+            	room.add('|raw|<b><font color="blue">' + user.name + ' is hosting the wheel.</font></b>');
+            	room.update();
+            	Economy.writeMoney(user.userid, Math.abs(room.wheel.wheel[room.wheel.wheel.length]) * room.wheel.wheelMulti * -1);
+            });
+        },                                                                          ///should parse the final number
```

Comment:
kyvn added that comment when we were first developing this my bad for not just removing in the first place